### ISSUE TITLE
mmcore install specific release

### ISSUE THREAD

**Issue by wl-stepp** [pymmcore-plus/pymmcore-plus#300]
Hello everybody,

I'm having problems using the --release input for mmcore install:
![image](https://github.com/pymmcore-plus/pymmcore-plus/assets/52414717/44559b02-7cad-4db8-a110-106213a3dcb0)

Not sure what the release should look like and the error message seems to be truncated. 

![image](https://github.com/pymmcore-plus/pymmcore-plus/assets/52414717/af4bf131-1a4b-4d1f-a119-d2e82b790c4c)

**Comment by wl-stepp** [comment]
I also think this doesn't really work. There is no "_win" in the folder name for me. Setting environment variable works however, so not urgent.

https://github.com/pymmcore-plus/pymmcore-plus/blob/78585577b50d12d30c9ae33c53e4fe447df651e6/src/pymmcore_plus/_util.py#L107C1-L116C58
```python
    # then look for an installation in this folder (from `pymmcore_plus.install`)
    sfx = "_win" if os.name == "nt" else "_mac"
    local_install = [
        p for p in PYMMCORE_PLUS_PATH.glob(f"**/Micro-Manager*{sfx}") if p.is_dir()
    ]
    if local_install:
        if return_first:
            logger.debug("using MM path from local install: %s", local_install[0])
            return str(local_install[0])
        full_list.extend([str(x) for x in local_install])
```

**Comment by tlambert03** [comment]
That second if clause is more or less obsolete at this point (it used to install directly into the pymmcore plus folder, and made a _win folder at that time.  Now it installs into proper user space.  Sorry about the useless error message, will check later today and get back to you.  You should also be able to manually download and install it into the user folder, and if you use `mmcore list` (and if you have at least one other install) it will show you that folder.  I also think it’s in the readme or docs somewhere.

**Comment by tlambert03** [comment]
ok, the error above is a windows specific issue with the `available_versions` parsing here:

https://github.com/pymmcore-plus/pymmcore-plus/blob/78585577b50d12d30c9ae33c53e4fe447df651e6/src/pymmcore_plus/install.py#L153

will try to post a fix later today

**Comment by tlambert03** [comment]
fixed by #301 ... but one note @wl-stepp, if you have multiple versions installed, there still isn't a good way to tell mmcore plus which one to select (short of using the environment variable).  We need to save a current selection somewhere, and add something like a `mmcore use` command to select
